### PR TITLE
Fixes #697 The Developer panel closes when and EditText is focused

### LINKED ISSUE #697: Developer options panel closed when an EditText is focused
## Hardware
All

## Steps to Reproduce
1. Open Settings
2. Open Developer Panel
3. Edit the Display Density setting
4. Click on the text field

## Current Behavior
The Developer Panel is closed.

## Expected Behavior
The Developer Panel shouldn't close and the keyboard should show.

## Possible Solution
Check if the new focused filed is a child of the developer panel when the global focus changes.

## Error Logs and Stack Traces
<details open>

```
```

</details>

### PR DESCRIPTION
**Pull request by keianhzo**
Fixes #697 #734 The Developer panel closes when and EditText is focused

### CODE REVIEW

File: app/src/common/shared/org/mozilla/vrbrowser/ui/settings/DoubleEditSetting.java [hunk 1]
```diff
@@ -35,12 +37,35 @@ private void initialize(Context aContext) {
...
         mEdit2.setSoundEffectsEnabled(false);
+        mEdit2.addTextChangedListener(mEdit2TextWatcher);
 
         mEdit2.setOnEditorActionListener(mInternalEditorActionListener);
     }
 
+    private TextWatcher mEdit2TextWatcher = new TextWatcher() {
+        @Override
+        public void beforeTextChanged(CharSequence s, int start, int count, int after) {
+
```
[the remark this answers is not in the document]
Comment (keianhzo) [reply app/src/common/shared/org/mozilla/vrbrowser/ui/settings/DoubleEditSetting.java]: We need to watch the text updates to update the EditText text color when selected/unselected. We are only using the onTextChanged method but we still need to implement the whole interface.

File: app/src/common/shared/org/mozilla/vrbrowser/ui/settings/DoubleEditSetting.java [hunk 2]
```diff
@@ -35,12 +37,35 @@ private void initialize(Context aContext) {
...
+        }
+
+        @Override
+        public void onTextChanged(CharSequence s, int start, int before, int count) {
+            mEdit1.selectAll();
+        }
+
+        @Override
+        public void afterTextChanged(Editable s) {
+
```
[the remark this answers is not in the document]
Comment (keianhzo) [reply app/src/common/shared/org/mozilla/vrbrowser/ui/settings/DoubleEditSetting.java]: Same as previous comment.

**Comment by keianhzo** [comment]
@cvan @caseyyee fixed all the above. Let me know if you see any other issues otherwise feel free to merge.

**Comment by caseyyee** [comment]
@keianhzo OK r+, looks good now!  Thanks for the quick turnaround!

**Pull request merged**